### ISSUE TITLE
Missing Polish characters while using polish.txt

### ISSUE THREAD

**Issue by TheRadziu** [ONElua/ONEMenu-for-PSVita#50]
OneMenu has the exact same issue as VitaShell with displaying Polish characters, more info on (still open/unresolved) VitaShell's issue:
https://github.com/TheOfficialFloW/VitaShell/issues/372

**Comment by ONElua** [comment]
Well, sorry for me badengrish.
Please send me a text and screenshot of how it should be displayed correctly...

**Comment by TheRadziu** [comment]
well, I can't post OneMenu's screenshot of how it should look properly (If I knew how to fix it, I'd push an commit instead of reposting the issue, isnt it :D?) but I can give you elaborated example. 
Original string: ```errorinstall =	"An error occurred while installing",```
Translated: ```errorinstall =	"Wystąpił błąd podczas instalacji",```
Expected: same as in translated string
Displayed result: ```Wyst_pił bł_d podczas instalacji```

or ```Zmień Nazwę``` is being displayed as ```Zmie_ Nazw_```

and here is a list of characters that being not displayed/replaced with `_` :
```ą, ć, ę, ł, ń, ó, ś, ź, ż```

Same happens under START button(system apps) 
![2018-05-24-211739](https://user-images.githubusercontent.com/7728545/40507674-130d7a9c-5f98-11e8-892f-d38a3a8f3acc.png)

They should be:
```
Menadżer zawartości
Przeglądarka
Wiadomości
Zdjęcia
```

**Comment by ONElua** [comment]
ok...
Thks

**Comment by TheRadziu** [comment]
AFAIK reason why it happens is that your homebrew uses one font/encoding (probably English by default?) for all languages, just loading strings from matching with current OS language `ux0:data/ONEMENU/lang/XYZ.txt` file if present, ignoring current system lang's font/dictionary/encoding/whatever. 

You might want to look into `sa0:/data/font/pgf/` for native vita font data, and this vitashell's commit which seems to load 'custom' fonts, which could be used to load respective pgf file for each language: 
https://github.com/TheOfficialFloW/VitaShell/blob/6753b26b910d8cc5525da6a2b272032b23bdef13/theme.c#L248

**Comment by ONElua** [comment]
Nope
https://github.com/ONElua/ONEMenu-for-PSVita/blob/master/ONEMenuVita/system/themes.lua#L116
I have the fix, but it only works with the pvf font. :(

Try with ths eboot.bin and change to PVF font
https://www.dropbox.com/s/c8ttfwjk2mbobwl/eboot.bin?dl=0

PVF Font (from system)
![2018-05-24-200830](https://user-images.githubusercontent.com/25811032/40547174-ca4d9b8e-5ff6-11e8-8b6f-0e54d118db27.png)

PGF Font (from system)
![2018-05-24-200941](https://user-images.githubusercontent.com/25811032/40547175-ca6b2064-5ff6-11e8-8ffd-e556f6f6e2e1.png)

**Comment by TheRadziu** [comment]
Yeah, It does work now, although I dont like how thin text is in comparison to PGF font, will try various system fonts to check if I can find closest working one

**Comment by ONElua** [comment]
We need to know if these characters really exist in the pgf font of the vita

**Comment by TheRadziu** [comment]
It doesnt seems so as I just tried couple of `ltnXX.pgf` fonts. I also extracted pgf files and you're right, THEY DO NOT HAVE polish characters. 

One option now would be forcing the font to one of these three: (imo looking best out of all pvf fonts) `ltn0.pvf` `ltn4.pvf` `ltn5.pvf` IF certain language file (for example polish.txt in that case, probably also german and other european languages that have their own characters) is being used.


Other option would be finding next closest font online that does support polish (and/or another language's) characters and force it if XYZ.txt language file is used.
